src/fluree/db/performance.clj

```diff
@@ -97,7 +97,7 @@
    "SELECT ?person ?fullName ?favNums\nWHERE {\n  ?person fd:person/fullName ?fullName;\n          fd:person/favNums  ?favNums;\n          fd:person/handle \"jdoe\", \"zsmith\".\n}"
    "SELECT DISTINCT ?horse ?horseLabel ?mother \n{\n    ?horse wdt:P31/wdt:P279* wd:Q726 .    \n    OPTIONAL{?horse wdt:P25 ?mother .}\n}"])
 
-(defn add-and-delete-data
```

Comment:
Did you intend to leave these functions commented out?

Reply:
Yeah, I didn't want to lose the form of the tests, but I also didn't want to replace them as part of this PR. I can nuke this file too if you think we should just redo it from scratch.